### DISCUSSION TITLE
Limits?

### DISCUSSION THREAD

**Discussion by g40** [duckdb/duckdb#5153] [General]
Excellent stuff, kudos and thanks.

Question: is there any hardwired limit to the numbers of columns/rows when importing from CSV?

The wiki does not mention any specific site for support. Is GitHub the place?

MTIA

**Comment by Alex-Monahan** [comment]
Hello! We try to avoid hard limits when we can! I don’t know about the CSV reader specifically though.
The DuckDB Discord is a good place for questions (link at the top of duckdb.org!). GitHub discussions are good for feature requests!

**Comment by g40** [comment]
Thanks @Alex-Monahan - not seeing a discord link. Have entries for documentation, blog , GitHub and contributing. That’s it. Thx++.

**Comment by Alex-Monahan** [comment]
> Thanks @Alex-Monahan - not seeing a discord link. Have entries for documentation, blog , GitHub and contributing. That’s it. Thx++.

Its in the purple banner at the top!
https://discord.com/invite/tcvwpjfnZx

**Comment by g40** [comment]
Aha. That explains it. If the banner link has been dismissed, it never reappears. This is true for Chrome on Windows 10 and Safari on iOS.

![image](https://user-images.githubusercontent.com/812250/199080311-4a8dec00-12e6-4eac-a02a-b30382a4f2dc.png)
